contracts/src/v0.8/ccip/test/DonIDClaimer.t.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {Ownable2Step} from "../../shared/access/Ownable2Step.sol";
+import {DonIDClaimer} from "../DonIDClaimer.sol";
+import {Test} from "forge-std/Test.sol";
+
+contract MockCapabilitiesRegistry {
+  uint32 private s_nextDonId;
+
+  constructor(
+    uint32 _initialDonId
+  ) {
+    s_nextDonId = _initialDonId;
+  }
+
+  function getNextDONId() external view returns (uint32) {
+    return s_nextDonId;
+  }
+}
+
+contract DonIDClaimerTest is Test {
+  DonIDClaimer private s_donIDClaimer;
+  MockCapabilitiesRegistry private s_mockRegistry;
+  address private s_owner = address(0x1);
+  address private s_deployer = address(0x2);
+  address private s_unauthorized = address(0x3);
+
+  function setUp() public {
+    vm.startPrank(s_owner);
+    s_mockRegistry = new MockCapabilitiesRegistry(100);
```

Comment:
Please use a constant for this 100, something like `INITIAL_CLAIM_ID` and refernce that in all tests to avoid magic numbers. That also means `INITIAL_CLAIM_ID + 1` instead of 101

Reply:
addressed here: https://github.com/smartcontractkit/chainlink-evm/pull/5/files